Sources/OpenAPIRuntime/Interface/Body.swift

```diff
@@ -0,0 +1,535 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the SwiftOpenAPIGenerator open source project
+//
+// Copyright (c) 2023 Apple Inc. and the SwiftOpenAPIGenerator project authors
+// Licensed under Apache License v2.0
+//
+// See LICENSE.txt for license information
+// See CONTRIBUTORS.txt for the list of SwiftOpenAPIGenerator project authors
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+//===----------------------------------------------------------------------===//
+
+import class Foundation.NSLock
+import protocol Foundation.LocalizedError
+import struct Foundation.Data  // only for convenience initializers
+
+/// The type representing a request or response body.
+public final class Body: @unchecked Sendable {
```

Comment:
Why is this a class? Also can we get rid of the `@unchecked Sendable` here and rather make ourselves a `LockedValueBox` abstraction that is conditionally `Sendable` when its contents are. This way we are making sure we really are `Sendable`

Reply:
> Why is this a class?

Because the body is inherently a reference type that cannot be freely copied.

> Also can we get rid of the @unchecked Sendable here and rather make ourselves a LockedValueBox abstraction that is conditionally Sendable when its contents are. This way we are making sure we really are Sendable.

Sure, that's an option that we can take as well, fortunately that won't affect the public API, so we can do that at any time.